.github/workflows/cr.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish Any Commit
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - run: corepack enable
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          cache: 'pnpm'
+      - run: pnpm install
+      - run: pnpm build
+      - run: pnpm dlx pkg-pr-new publish './dist' --compact --template './examples/*'
```

Comment:
Can you add a newline at the end?

BTW, we may want a simpler example like `examples/counter`. `examples/demo` might be too big for testing.
If you don't have time, let me know so that I can take it.

Reply:
@dai-shi 

> Can you add a newline at the end?

In this case, i can fix it in new PR ASAP

Reply:
We do have this anyway: https://github.com/pmndrs/jotai/pull/2862
Let's see if @Aslemammad responds in 24 hours.

@sukvvon Would you be interested in creating a simple counter app in `examples/counter`?

Reply:
@dai-shi  i knew that counter app is already exist in `examples/demo`, could you explain your purpose?

Reply:
demo has many dependencies for visual effects. for testing purpose with pkg.pr.new, a simpler counter example would be easier.

btw, we also have an issue related with examples in valtio: https://github.com/pmndrs/valtio/discussions/1012

Reply:
> BTW, we may want a simpler example like examples/counter. examples/demo might be too big for testing.

@dai-shi do you wanna separating it?

Reply:
feel free to tackle it @sukvvon i'll meanwhile finish this PR reviews and jotai's PR too.

Reply:
> @sukvvon Would you be interested in creating a simple counter app in examples/counter?

In this case, I can write the logic, but it’s hard for me to clearly understand what is desired stylistically, so I think it would be better for @dai-shi to handle it. 😄

Reply:
No problem. I can handle it, but I don't do any styling.